### ISSUE TITLE
Deep transclusion of files

### ISSUE THREAD

**Issue by it-spiderman** [inblockio/mediawiki-extensions-Aqua#322]
Same issue as https://github.com/inblockio/mediawiki-extensions-Aqua/issues/291, but for file transclusions.

**Comment by it-spiderman** [comment]
Fixed in `main`, with https://github.com/inblockio/mediawiki-extensions-Aqua/commit/7630a20d211afb1bdf2ae38554b52f07caf95bd7, but its a quick fix, as its very very hard to know which files are direct transclusions, even harder than for templates.

**Comment by FantasticoFox** [comment]
Please advance this code to be an option which can be activated or deactivated in the DA-CONFIG page.
Please also do a strong warning, that his can only be activated once, and then can't be deactivated as pages would break if activated again later.

So the option can only be enabled 'permanently once' for the PKC.

Before starting with this, this requires a short discussion how this exactly functions and what are the implications of this feature. Please write those implications upfront here @it-spiderman 

It might very well be, that this can be solved on the verification level to avoid manipulation of sub-ressources.

**Comment by FantasticoFox** [comment]
This has become a won't fix.
